src/main.h

```diff
@@ -8,6 +8,7 @@ See the file LICENSE for details.
 #define MAIN_H
 
 extern struct fs_dirent *root_directory;
+extern struct fs_dirent *current_directory;
```

Comment:
Should be a property of the current process, not a global.

Reply:
This is changed in my syscall branch, this was just a temporary way to be able to run terminal-like commands in kshell.

Reply:
ok